go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,61 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the NativeMessage whitelists for root user.
+	Root string
+	// Home is the path of the NativeMessage whitelists for regular users.
+	Home string
+	// Overlay is an optional path to prefix all paths with. Useful for
+	// generating whitelists for Linux distributio packaging.
+	Overlay string
```

Comment:
It's a little bit funky to make Overlay a member on the whitelist path here. Maybe it should be a parameter to `Install` and `path`? Definitely not something I feel strongly about though.